jubatus/server/framework/mixer/linear_mixer.cpp

```diff
@@ -391,17 +398,23 @@ void linear_mixer::stabilizer_loop() {
         }
       }
 
+      lk.lock();
```

Comment:
The lock may be already taken.
It's better to keep code be explicit; even if it is harmless.